docs/source/examples/interactive-development.rst

```diff
@@ -0,0 +1,170 @@
+Start a Development Cluster
+===========================
+
+SkyPilot makes the interactive development very easy on clouds. It helps you:
+
+#. Launch: a cluster with GPU or any specific requirement with a single command.
+#. Connect: Quickly connect to the cluster with the cluster name.
```

Comment:
This ordering is different from sections below?

Reply:
Moved autostop earlier as it might be more like a value add : )